.github/workflows/linux.yml

```diff
@@ -43,6 +43,11 @@ jobs:
             cxxflags: -stdlib=libc++
             os: ubuntu-20.04
             install: sudo apt install libc++-11-dev libc++abi-11-dev
+          - cxx: g++-11
+            build_type: Debug
+            std: 20
+            os: ubuntu-20.04
+            install: sudo apt install g++-11
```

Comment:
nit: I suggest grouping gcc configs together by moving this entry right after `g++-10`.

Reply:
I was thinking about replacing `g++-10 -std=c++20` config with the new one `g++-11 -std=c++20` to get less configs, what do you think about that?

Reply:
Sounds good to me.

Reply:
Replaced and moved to the other GCC configs